orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/Front50Service.groovy

```diff
@@ -129,6 +130,18 @@ interface Front50Service {
   @POST("/serviceAccounts")
   Response saveServiceAccount(@Body ServiceAccount serviceAccount)
 
+  @GET("/deliveries/{id}")
+  Response getDeliveryConfig(@Path("id") String id)
```

Comment:
How come we don't type the response, e.g. `Response<com.netflix.spinnaker.front50.model.delivery.Delivery>`

Reply:
I think that typed responses is a feature of retrofit 2, and we're on 1.9. I agree that typing of responses would be better, but we would have to upgrade Orca.

Reply:
Responses can be typed. Use `Response` if you need access to additional information such as headers or response status.